source/solutions-library/digital-underwriting.txt

```diff
@@ -0,0 +1,245 @@
+         :alt: An illustration shows on how to create a data processing pipeline  
+  
+      Figure 3: Creating a data processing pipeline  
+  
+   .. step:: Automating insurance premium calculations with a machine learning model  
+  
+      The decision-processing component of this example consists of a  
+      scheduled trigger that collects the necessary data and posts the  
+      payload to a Databricks ML Flow API endpoint. (The model was  
+      previously trained using the MongoDB Spark Connector on  
+      Databricks.) It then waits for the model to respond with a  
+      calculated premium based on the miles driven by a given customer  
+      in a month. Then the scheduled trigger updates the  
+      ‘customerPolicy’ collection to append a new monthly premium  
+      calculation as a new subdocument within the ‘monthlyPremium’  
+      array.  
+  
+      See the following Github repos to create the data processing pipeline:  
+  
+      - Step 4 `Setup a calculate premium trigger <https://github.com/mongodb-industry-solutions/Digital-Underwriting-Usage-Based-Insurance/blob/main/src/CalculatePremiumTrigger.md>`__  
+  
+      - Step 5 `Setup the Databricks connection <https://github.com/mongodb-industry-solutions/Digital-Underwriting-Usage-Based-Insurance/blob/main/src/DatabricksConfiguration.md>`__  
+  
+      - Step 6 `Write the machine learning model prediction to MongoDB <https://github.com/mongodb-industry-solutions/Digital-Underwriting-Usage-Based-Insurance/blob/main/src/Prediction.md>`__  
+  
+      .. figure:: /includes/images/industry-solutions/Writing Fig4.svg  
+         :figwidth: 1200px  
+         :alt: Automating Calculations with Machine Learning Model  
+  
+      Figure 4: Automating Calculations with Machine Learning Model  
+
+   .. step:: Near-real-time insights of insurance premium changes over time
+
+      Once the monthly premium calculations have been appended, it’s
+      easy to set up Atlas Charts to visualize your newly calculated
+      usage-based premiums. Configure different charts to see how
+      premiums have changed over time to discover patterns.
+      
+Technologies and Products Used
+------------------------------
+
+MongoDB modern, multi-cloud database platform:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- `Atlas Database <https://www.mongodb.com/atlas/database>`__
+- `Aggregation Pipelines <https://www.mongodb.com/docs/v7.0/core/aggregation-pipeline/>`__
+- `Materialized Views <https://www.mongodb.com/docs/v5.0/core/materialized-views/>`__
+- `Time Series <https://www.mongodb.com/time-series>`__
+- `MongoDB Spark Connector <https://www.mongodb.com/products/spark-connector>`__
+- `Atlas Charts <https://www.mongodb.com/products/charts>`__
+- `Atlas App Services <https://www.mongodb.com/atlas/app-services>`__
+  -`Triggers <https://www.mongodb.com/docs/atlas/app-services/triggers/>`__
+  -`Functions <https://www.mongodb.com/docs/atlas/app-services/functions/>`__
```

Comment:
This also has a slight formatting issue in staging. I think we need to add a space between the bullet and the text:

```suggestion
- `Atlas App Services <https://www.mongodb.com/atlas/app-services>`__
  - `Triggers <https://www.mongodb.com/docs/atlas/app-services/triggers/>`__
  - `Functions <https://www.mongodb.com/docs/atlas/app-services/functions/>`__
```